app/views/SettingsView/index.js

```diff
@@ -1,236 +1,108 @@
-		} = this.state;
-		const { useMarkdown } = this.props;
+		const { server, useMarkdown } = this.props;
 		return (
-			<KeyboardView
-				contentContainerStyle={sharedStyles.container}
-				keyboardVerticalOffset={128}
-			>
+			<SafeAreaView style={styles.container}>
 				<StatusBar />
 				<ScrollView
-					contentContainerStyle={sharedStyles.containerScrollView}
-					testID='settings-view-list'
 					{...scrollPersistTaps}
+					contentContainerStyle={styles.contentContainer}
+					showsVerticalScrollIndicator={false}
 				>
-					<SafeAreaView style={sharedStyles.container} testID='settings-view' forceInset={{ bottom: 'never' }}>
-						<RNPickerSelect
-							items={languages}
-							onValueChange={(value) => {
-								this.setState({ language: value });
-							}}
-							value={language}
-							placeholder={placeholder}
-						>
-							<RCTextInput
-								inputRef={(e) => { this.name = e; }}
-								label={I18n.t('Language')}
-								placeholder={I18n.t('Language')}
-								value={this.getLabel(language)}
-								testID='settings-view-language'
-							/>
-						</RNPickerSelect>
-						<View style={sharedStyles.alignItemsFlexStart}>
-							<Button
-								title={I18n.t('Save_Changes')}
-								type='primary'
-								onPress={this.submit}
-								disabled={!this.formIsChanged()}
-								testID='settings-view-button'
-							/>
-						</View>
-						<View style={styles.separator} />
-						<View style={styles.swithContainer}>
-							<Text style={styles.label}>{I18n.t('Enable_markdown')}</Text>
-							<Switch
-								value={useMarkdown}
-								onValueChange={this.toggleMarkdown}
-								onTintColor={COLOR_SUCCESS}
-								tintColor={isAndroid ? COLOR_DANGER : null}
-							/>
-						</View>
-						<Loading visible={saving} />
-						<Toast ref={toast => this.toast = toast} />
-					</SafeAreaView>
+
+					<Button title={I18n.t('Contact_us')} onPress={this.sendEmail} showActionIndicator />
+					{this.renderSeparator()}
+					<Button title={I18n.t('Language')} onPress={this.navigateToRoom('LanguageView')} showActionIndicator />
+					{this.renderSeparator()}
+					<Button title={I18n.t('Theme')} showActionIndicator disable />
+					{this.renderSeparator()}
+					<Button title={I18n.t('Share_this_app')} showActionIndicator disable />
+
+					{this.renderSectionSeparator()}
+
+					<Button title={I18n.t('License')} onPress={this.openLink(LICENSE_LINK)} showActionIndicator />
+					{this.renderSeparator()}
+					<Button title={I18n.t('Version_no', { version: getReadableVersion })} />
+					{this.renderSeparator()}
+					<Button title={I18n.t('Server_version', { version: server.version })} subTitle={`${ server.server.split('//')[1] }`} />
+
+					{this.renderSectionSeparator()}
+
+					<ButtonWithSwitch title={I18n.t('Enable_markdown')} value={useMarkdown} onValueChange={this.toggleMarkdown} />
+					{this.renderSeparator()}
+					<Button title={I18n.t('Crash_report_disclaimer')} disable />
```

Comment:
Why is this a button if it's not touchable?
Even if it's disabled, it shouldn't be a button.
You should use semantics here.

Natively the device will build a button and apply a disable prop.
It should be a simple view.